extras/test/unit/tests/TestCalculateTimeOnAir.cpp

```diff
@@ -0,0 +1,75 @@
+#include <boost/test/unit_test.hpp>
+#include "modules/SX126x/SX126x.h"
+#include "modules/SX127x/SX127x.h"
+#include "modules/SX128x/SX128x.h"
+#include "modules/LR11x0/LR11x0.h"
+
+// Testable wrappers
+class SX127xTestable : public SX127x {
```

Comment:
Is this wrapper really necessary? The other classes do not have it, so this might be a sign there's something wrong in the SX127x class.

Reply:
Yes, but only because the functions `SX127x::reset()` and `SX127x::errataFix()` are pure virtual. Should I replace them with an empty function and remove the wrapper?

Reply:
Implemented in the last commit.